components/match2/wikis/apexlegends/match_group_input_custom.lua

```diff
@@ -40,7 +40,6 @@ local NP_STATUSES = {'skip', 'np', 'canceled', 'cancelled'}
 local NOT_PLAYED_SCORE = -1
 local SECONDS_UNTIL_FINISHED_EXACT = 30800
 local SECONDS_UNTIL_FINISHED_NOT_EXACT = 86400
-local EPOCH_TIME = '1970-01-01T00:00:00+00:00'
```

Comment:
can we replace this easily with `DateExt.defaultDateTime = '1970-01-01 00:00:00'` or do we need another one for that?